website/content/docs/nia/configuration.mdx

```diff
@@ -190,6 +191,7 @@ Service registration requires that the [Consul token](/docs/nia/configuration#co
 | --------- | -------- | ---- | ----------- | ------- |
 | `enabled` | Optional | boolean | Enables CTS to register itself as a service with Consul.| `true` |
 | `service_name` | Optional | string | The service name for CTS. | `Consul-Terraform-Sync` |
+| `address` | Optional | string | The IP address or hostname for CTS. | IP address of the agent node |
```

Comment:
Just want to check my understanding of "agent node" - does "agent" here refer to the Consul agent (not CTS right)?

Reply:
Yes, it's the Consul agent here! Let me change this to specify `the Consul agent node`.